### PR TITLE
postgresql@17 17.2

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by daeho-ro** [comment]
```
  configure: error: could not find function 'gss_store_cred_into' required for GSSAPI
```

Same error happen in 
- https://github.com/Homebrew/homebrew-core/pull/197716

File: Formula/p/postgresql@17.rb [hunk 1]
```diff
@@ -181,8 +181,8 @@ def caveats
     system bin/"initdb", testpath/"test" unless ENV["HOMEBREW_GITHUB_ACTIONS"]
     [bin/"pg_config", HOMEBREW_PREFIX/"bin/pg_config-#{version.major}"].each do |pg_config|
       assert_equal "#{HOMEBREW_PREFIX}/share/#{name}", shell_output("#{pg_config} --sharedir").chomp
-      assert_equal "#{HOMEBREW_PREFIX}/lib/#{name}", shell_output("#{pg_config} --libdir").chomp
-      assert_equal "#{HOMEBREW_PREFIX}/lib/#{name}", shell_output("#{pg_config} --pkglibdir").chomp
+      assert_equal lib.to_s, shell_output("#{pg_config} --libdir").chomp
+      assert_equal lib.to_s, shell_output("#{pg_config} --pkglibdir").chomp
```
Comment (chenrui333) [inline Formula/p/postgresql@17.rb:185]: might need to understand why this got changed, seems a bit fishy

Comment (daeho-ro) [reply Formula/p/postgresql@17.rb:185]: Indeed, it feels like sudden change and we have to check it.

Comment (chenrui333) [reply Formula/p/postgresql@17.rb:185]: gonna adopt something similar as https://github.com/Homebrew/homebrew-core/pull/199616

**Review by daeho-ro — APPROVED** [review]

**Pull request merged**